samples/ios/app/glean-sample-appUITests/BaselinePingTest.swift

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Glean
+import Swifter
+import XCTest
+
+// swiftlint:disable force_cast
+// REASON: Used in below test cases to cause errors if data is missing
+class BaselinePingTest: XCTestCase {
+    var app: XCUIApplication!
+    var expectation: XCTestExpectation?
+    var lastPingJson: [String: Any]?
+    var server: HttpServer?
+
+    override func setUp() {
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // UI tests must launch the application that they test.
+        // Doing this in setup will make sure it happens for each test method.
+        app = XCUIApplication()
+    }
+
+    override func tearDown() {
+        server?.stop()
```

Comment:
Is this being called after each test? If so, should we call `setupServer` in `setUp`? (or comment on why this is not being done)

Reply:
This is by now mostly a left-over of my previous code.
I'm refactoring the server function in #444, would you okay with letting this slip here and I move around the `server.stop` over in the follow up?

Reply:
Yup! That's fine, thanks!